### ISSUE TITLE
Unexpected task completions on nightly reset

### ISSUE THREAD

**Issue by lukefernandez** [amitness/habitist#27]
Issue: Every morning (I set the cron to run at midnight), I was noticing that I already had a large number of completed tasks for the day. Digging into the code a bit, I realized that tasks I didn't complete the previous day were being marked complete when the count was reset to zero. That isn't exactly what I would expect. If I did not complete a task, I would want it to reset to zero, but I wouldn't want it to be marked as complete.

Resolution: Instead of making a call to item.update_date_complete, make a call to item.update so that the count resets to zero but the item is not marked as complete.

This change has been proposed in PR #26.

Let me know if you need additional context! Thanks for putting this awesome project together!

**Comment by amitness** [comment]
@lukefernandez Thank you. Can you test this in your fork for a few days and let me if this works correctly?

I reviewed old commits and saw that I had switched from `update` to `update_date_complete` [here](https://github.com/amitness/habitist/commit/d67e1425cabffe4badd6d68f75d93174e17a873d). But can't recall the exact reason. I guess it was due to some issue with the date not being set correctly when using `update`.

**Comment by lukefernandez** [comment]
Yeah, it seems to be working on my end, but I'll think of some more rigorous test cases and upload the results here in the next few days. Thanks!
